Box.V2/Box.V2.csproj

```diff
@@ -1,383 +1,45 @@
-<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
+<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
-    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
+    <PackageId>Box.V2</PackageId>
+    <TargetFramework>net462</TargetFramework>
+    <Version>5.8.0</Version>
+    <Description>Box SDK for v2 of the Box API. The SDK is targeting .NET Framework 4.6.2</Description>
+    <Authors>Box Inc.</Authors>
+    <RepositoryUrl>https://github.com/box/box-windows-sdk-v2.git</RepositoryUrl>
+    <RepositoryType>git</RepositoryType>
+    <PackageProjectUrl>https://github.com/box/box-windows-sdk-v2</PackageProjectUrl>
+    <PackageLicenseExpression>Apache-2.0</PackageLicenseExpression>
+    <PackageReleaseNotes>See https://github.com/box/box-windows-sdk-v2/blob/main/CHANGELOG.md#580-2024-07-22</PackageReleaseNotes>
+    <PackageTags>Box V2 SDK Platform Enterprise Collaboration Storage File Management</PackageTags>

```

Comment:
In `Box.V2.Core/Box.V2.Core.csproj` file in `PackageTags` attribute the tags are separated by semicolon `;`.
Does it matter?

Shouldn't we add here `AssembyTitle` like in `Box.V2.Core.cproj`?

Reply:
Looks like we can safely remove:
- `AssemblyName` - 
> Specify the full name and extension of the file you want to create. If you don't specify the name of the output file, MSBuild uses the name of the project to specify the name of the output assembly. 
Taken from https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/compiler-options/output#outputassembly

- `AssemblyTitle` - it's inferred from the `AssemblyName`.

After removing these props, looks like both title and name have proper values 

<img width="562" alt="Screenshot 2025-05-29 at 11 49 15" src="https://github.com/user-attachments/assets/b8682951-c66a-4c89-b319-e99f72e9faf8" />